### PR TITLE
SAJ integration doc update with UI configuration

### PR DESCRIPTION
**Pull request by sanyatuning**
## Proposed change

SAJ integration doc update with UI configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/41198
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x]  This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/saj.markdown [hunk 1]
```diff
@@ -19,7 +21,15 @@ The WiFi module requires a username and password for authentication where the et
...
-To enable this sensor, add the following lines to your `configuration.yaml` file:
+1. In the **Configuration** > **Integrations** menu, click the `+ ADD INTEGRATION` button on the lower-right of the screen.
+2. Select **SAJ Solar Inverter** from the pop-up menu.
+3. Complete the dialog box and click `SUBMIT`. 
+
+Note: You should set up fixed/static IP address for the inverter.
+
+## Manual configuration
+
+If you prefer to set up the integration in `configuration.yaml`, add the following lines:
```
Comment (frenck) [inline source/_integrations/saj.markdown]: The manual configuration method should be deprecated and removed from documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/saj.markdown (hunk 1 above)
Comment (sanyatuning) [reply source/_integrations/saj.markdown]: So... Should I mark it as deprecated or should I remove it?

Comment (frenck) [reply source/_integrations/saj.markdown]: The code should throw deprecation warnings, in the documentation, it should be removed.

Comment (sanyatuning) [reply source/_integrations/saj.markdown]: Thanks!

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed

**Pull request closed without merging**